.devcontainer/devcontainer.json

```
@@ -6,7 +6,7 @@
 		"dockerfile": "Dockerfile",
 		"args": {
 			"VARIANT": "hugo",
-			"VERSION": "0.82.1",
+			"VERSION": "0.92.0",
```

Actually we're up to 96 now:

```suggestion
			"VERSION": "0.96.0",
```